tests/raftstore_cases/test_compact_after_delete.rs

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use tikv::storage::{CF_DEFAULT, CF_WRITE};
+use tikv::storage::{CF_DEFAULT, CF_LOCK};
```

Comment:
Because i use the same config as we use in production which has `prefix_extractor` for CF_WRITE, but the test key doesn't has ts in the tail, this will cause error.